app/main.py

```diff
@@ -1,13 +1,22 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "errors": [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "passed" if len(errors) == 0 else "failed"}
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+
+    return [format_single_linter_file(file_path, errors)
+            for file_path, errors in linter_report.items()]
```

Comment:
fix indentation

Reply:
I can't fix in another way, because when I try to check my task via flake8 app, there is the error - 
app\main.py:23:80: E501 line too long (104 > 79 characters)

![image](https://user-images.githubusercontent.com/120219525/208638772-0ea7e368-84f8-437e-a653-400a106f77d2.png)

Reply:
try to use `black`

Reply:
Help! I read about Black, I understand that i need  to change properties from PEP8 to Black, but I don't know how. I haven't preferences in my Menubar (Edit)
![image](https://user-images.githubusercontent.com/120219525/208643602-0fe4b771-9780-4b35-b676-333b5a33ddef.png)